protocol/x/sending/keeper/transfer.go

```diff
@@ -112,9 +112,6 @@ func (k Keeper) ProcessDepositToSubaccount(
 			float32(msgDepositToSubaccount.Quantums),
 			[]gometrics.Label{
 				metrics.GetLabelForIntValue(metrics.AssetId, int(msgDepositToSubaccount.AssetId)),
-				metrics.GetLabelForStringValue(metrics.SenderAddress, msgDepositToSubaccount.Sender),
```

Comment:
@tqin7 Can you confirm that we don't have any monitoring/alerting based on these labels?